schemas/input/csl-data.json

```diff
@@ -4,425 +4,477 @@
   "$id": "https://resource.citationstyles.org/schema/latest/input/json/csl-data.json",
   "type": "array",
   "items": {
-    "type": "object",
-    "properties": {
-      "type": {
-        "type": "string",
-        "enum": [
-          "article",
-          "article-journal",
-          "article-magazine",
-          "article-newspaper",
-          "bill",
-          "book",
-          "broadcast",
-          "chapter",
-          "classic",
-          "collection",
-          "dataset",
-          "document",
-          "entry",
-          "entry-dictionary",
-          "entry-encyclopedia",
-          "figure",
-          "graphic",
-          "hearing",
-          "interview",
-          "legal_case",
-          "legislation",
-          "manuscript",
-          "map",
-          "motion_picture",
-          "musical_score",
-          "pamphlet",
-          "paper-conference",
-          "patent",
-          "performance",
-          "periodical",
-          "personal_communication",
-          "post",
-          "post-weblog",
-          "regulation",
-          "report",
-          "review",
-          "review-book",
-          "software",
-          "song",
-          "speech",
-          "standard",
-          "thesis",
-          "treaty",
-          "webpage"
-        ]
-      },
-      "id": {
-        "type": ["string", "number"]
-      },
-      "categories": {
-        "type": "array",
-        "items": {
+    "$ref": "#/definitions/bibitem"
+  },
+  "definitions": {
+    "bibitem": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "article",
+            "article-journal",
+            "article-magazine",
+            "article-newspaper",
+            "bill",
+            "book",
+            "broadcast",
+            "chapter",
+            "classic",
+            "collection",
+            "dataset",
+            "document",
+            "entry",
+            "entry-dictionary",
+            "entry-encyclopedia",
+            "figure",
+            "graphic",
+            "hearing",
+            "interview",
+            "legal_case",
+            "legislation",
+            "manuscript",
+            "map",
+            "motion_picture",
+            "musical_score",
+            "pamphlet",
+            "paper-conference",
+            "patent",
+            "performance",
+            "periodical",
+            "personal_communication",
+            "post",
+            "post-weblog",
+            "regulation",
+            "report",
+            "review",
+            "review-book",
+            "software",
+            "song",
+            "speech",
+            "standard",
+            "thesis",
+            "treaty",
+            "webpage"
+          ]
+        },
+        "id": {
```

Comment:
Technical issue: `id` is required, but we don't need it for the embedded form.

Reply:
Hmm, should we make a `standard-bibitem` definition that is `bibitem` plus required `id`, whereas `related-bibitem` is either `id` _or_ `bibitem`?

Reply:
Should be fixed now. Thanks for the tip.